vipps-recurring-api.md

```diff
@@ -536,36 +559,104 @@ when the product is shipped.
 
 ### Campaigns
 
-A campaign in recurring is a period where the price is lower than usual, and
-this is communicated to the customer with the original price shown for comparison.
+Campaigns can be used to lower the price of the agreement for a limited amount of time.
+Recurring supports 4 different campaign types : price campaign, period campaign, event campaign, full flex campaign, see more about the different campaign types in the table below. This is communicated to the customer with the original price shown for comparison.
```

Comment:
" :"  should not have the space before.

Reply:
I would have said: Recurring supports 4 different campaign types: price campaign, period campaign, event campaign, and full flex campaign. See more about the different campaign types in the table below.

Reply:
Done 👍